add validator FAQ link to landing page

georgeweiler opened the pull request.

CarlBeek commented:
I'd like to keep this as an internal link if possible. Can we use the template that @Battenfield proposed in #89?

georgeweiler commented:
@CarlBeek Understood. I will create an FAQ route with launchpad styling and copy the content from that FAQ over.

georgeweiler commented:
updated the links to primary color using the new `withArrow` prop, and added `/faq` link.
<img width="596" alt="Screen Shot 2020-07-21 at 10 07 14 AM" src="https://user-images.githubusercontent.com/15269424/88085053-2c4ce500-cb3a-11ea-90d0-ea643b292358.png">

CarlBeek commented on src/pages/Landing/Phases/index.tsx:
```diff
@@ -54,15 +55,25 @@ const phases: phase[] = [
+  {
+    title: 'Validators FAQ',
+    subTitle:
+      'Check out the FAQ page for more information about the roles and responsibilities of ETH 2 network validators.',
```
```suggestion
      'Check out the FAQ page for more information about the roles and responsibilities of eth2 network validators.',
```
